tests/proxy_models/tests.py

```diff
@@ -105,6 +105,17 @@ def test_proxy_included_in_ancestors(self):
         with self.assertRaises(Person.MultipleObjectsReturned):
             StatusPerson.objects.get(id__lt=max_id + 1)
 
+    def test_multiple_abc(self):
+        """
+        Proxy cannot have multiple abstract base classes
+        """
+        def build_multiple_abc():
```

Comment:
I could be missing something, but I don't see a need for the function.
```
msg = "Proxy model 'MultipleAbstract' has more than one non-abstract model base class."
with self.assertRaisesMessage(TypeError, msg):
    class MultipleAbstract(User, Person):
        class Meta:
            proxy = True
```
should be okay, I think? If so, we can make a separate commit to clean up the existing tests (and make them use `assertRaisesMessage` instead of just `assertRaises` too.